proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -6,41 +6,41 @@ import "google/protobuf/wrappers.proto";
 
 option go_package = "github.com/Traceableai/agent-config/gen/go/v1";
 
-// Parent configuration object for Opa & Blocking Config
-message AgentConfig {
-    Opa opa = 1;
-    BlockingConfig blocking_config = 2;
+message LibtraceableConfig {
```

Comment:
Backwards compatibility and code generation

Reply:
For code gen, I just ran `buf generate` on my mac?